dist/plugin.json

```
@@ -84,6 +84,20 @@
           "content": "application/json"
         }
       ]
+    },
+    {
+      "path": "telemetry_labels",
+      "url": "https://api.lightstep.com/public/v0.2/{{ .JsonData.orgName}}/projects/{{ .JsonData.projectName}}/telemetry_labels",
```

I think this is missing a `/telemetry`
```suggestion
      "url": "https://api.lightstep.com/public/v0.2/{{ .JsonData.orgName}}/projects/{{ .JsonData.projectName}}/telemetry/telemetry_labels",
```